website/www/site/content/en/documentation/transforms/python/overview.md

```diff
@@ -82,4 +82,6 @@ limitations under the License.
   most useful for adjusting parallelism or preventing coupled failures.</td></tr>
   <tr><td><a href="/documentation/transforms/python/other/windowinto">WindowInto</a></td><td>Logically divides up or groups the elements of a collection into finite
   windows according to a function.</td></tr>
+  <tr><td><a href="/documentation/transforms/python/other/waiton">WaitOn</a></td><td>Delays processing of a PCollection until other PCollections have finished processing.</td></tr>
```

Comment:
Could you please add Wait/WaitOn to the sidebard on the left hand side of the page? Thanks!
See (e.g.) Transform Catalog > Java > Other. Currently rendered version (available at https://apache-beam-website-pull-requests.storage.googleapis.com/35647/documentation/transforms/java/other/wait/index.html ) doesn't have it.